typings/index.d.ts

```diff
@@ -2596,5 +2615,23 @@ declare module 'discord.js' {
 	type CloseEvent = { wasClean: boolean; code: number; reason: string; target: WebSocket; };
 	type ErrorEvent = { error: any, message: string, type: string, target: WebSocket; };
 
+	enum ChannelType {
```

Comment:
discord.js does not export such an enum.
Maybe it's possible to declare it to be "type-only", so you can not use it as a value.

Reply:
I'd love to know why TS exports this automatically, or how I could declare it without it being exported... Maybe moving it outside the `declare module`?